templates/quickstart-jira-dc-with-vpc.template.yaml

```
@@ -0,0 +1,728 @@
+---
+# Bring up diffs between this file and the BitbucketDataCenter.template.yaml, ConfluenceDataCenter.template.yaml, and CrowdDataCenter.template.yaml
+# As a rule, we should work to minimize our diffs between these files as best we can, so that future changes are easy to make across all supported CloudFormation templates.
+# Using YAML as our file format will allow us to put a block comment at the top of the file saying exactly this.
+AWSTemplateFormatVersion: 2010-09-09
+Description: Atlassian Jira Data Center
```

This should reflect the template structure. In this case, the Jira is spun up with VPC as well.
```suggestion
Description: Atlassian Jira Data Center with VPC
```